scripts/rc.py

```diff
@@ -204,6 +234,9 @@ def dp_main(args):
     elif args.mv_local:
         print("Attempting to move files locally")
         mv_local(args.mv_local)
+    elif args.mv_aws:
+        print("Attempting to move files AWS")
```

Comment:
This should be something like "Attempting to move files to/from AWS bucket"

Reply:
Note this might not be enough if an S3 bucket has KVS key encryption.